src/tools/interop/interop.cpp

```diff
@@ -435,6 +445,16 @@ class InteropConnection {
                     sizeof(Settings),
                     &Settings));
     }
+    bool SetResumptionTicket(const uint8_t* Ticket, uint32_t TicketLength) {
+        return
+            QUIC_SUCCEEDED(
+                MsQuic->SetParam(
+                    Connection,
+                    QUIC_PARAM_LEVEL_CONNECTION,
+                    QUIC_PARAM_CONN_RESUMPTION_STATE,
```

Comment:
I don't like this name. I want to rename it to something like "RESUMPTION TICKET". Thoughts?

Reply:
Yeah, might be good.

Reply:
It should probably just match the recv callback event for it.